src/api/dataSource/NodeDataSource.ts

```diff
@@ -218,13 +238,17 @@ export class NodeDataSource {
     initArguments: string,
   ): Promise<boolean> {
     try {
+      const authHeaders: AxiosHeader | null = await createAuthHeader(
+        `${applicationId}${initFunction}${initArguments}`,
```

Comment:
This is payload `${applicationId}${initFunction}${initArguments}` ?

Reply:
What do you propose to add insead? Payload is just any string signed so idea was to sign whatever the props of the functions are.

Reply:
We are signing params which you can then later verify to make sure payload is not tempered. So just convert it to object which can be parsed later instead of combined string